pkg/http.go

```diff
@@ -23,6 +23,12 @@ type Response struct {
 	Type   string `json:"type,omitempty"`
 }
 
+var version string // This will be injected at build time
```

Comment:
let's also set a `var commitSHA string` so we can easily link it back to a commit

Reply:
// This will be injected at build time
var (
	version   string
	commitSHA string
)